apps/docs/content/tokens/semantic/typography.mdx

```diff
@@ -9,6 +9,22 @@ import tokens from '../../../datas/tokens.json';
 
 Typography tokens are composite tokens, meaning they are made up of multiple other tokens. This allows us to create a consistent typographic scale across our products.
 
+## Guidelines
+
+### Working with typography
+
+#### Use the bounding box to align with grid
+
+The bounding box is the vertical height of the text and is defined by the line height. That value of the line height is critical to make sure text is aligned to the 8px grid of the interface.
+
+### Line length should be between 40 to 60 characters.
+
+Too shorts lines of text makes the eyes strain while long lines makes it hard to concentrate. As a way to give users the best reading experience, aim for lines of text between 40 to 60 characters, including spaces. If your text doesn’t fit this rule, review the content length, the font size or even information hierarchy.
+
+### Align text using baseline
+
+When aligning different text boxes together horizontally, use their baselines as guide instead of their true centre for a more harmonious look.
```

Comment:
```suggestion
When aligning different text boxes together horizontally, use their baselines as a guide instead of their true center for a more harmonious look.
```